apps/stats-web/.eslintrc.cjs

```
@@ -0,0 +1,18 @@
+const path = require("path");
```

ESLint

[error] 'path' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'path' is assigned a value but never used.

(no-unused-vars)